config.example.js

```diff
@@ -74,13 +74,13 @@ module.exports = {
     SQ_EXTENSION_BLACKLIST: ["exe"],
 
     // The URL of your tracker site.
-    SQ_BASE_URL: "https://sqtracker.dev",
+    SQ_BASE_URL: "http://localhost",
```

Comment:
Please revert changes to this file. This file is to serve as a production example, using localhost here is incorrect